Error while checking build/O2/fullCI for 789f8a0b64e690a4c090fed621ceeb586ac04dc6 at 2023-11-22 11:51:
```
## sw/BUILD/O2-latest/log
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
/sw/slc8_x86-64/fmt/10.1.1-1/include/fmt/core.h:1576:63: error: 'fmt::v10::detail::type_is_unformattable_for<const o2::header::Descriptor<4>, char> _' has incomplete type
/sw/slc8_x86-64/fmt/10.1.1-1/include/fmt/core.h:1580:7: error: static assertion failed: Cannot format an argument. To make type T formattable provide a formatter<T> specialization: https://fmt.dev/latest/api.html#udt
/sw/slc8_x86-64/fmt/10.1.1-1/include/fmt/core.h:1576:63: error: 'fmt::v10::detail::type_is_unformattable_for<const o2::header::Descriptor<16>, char> _' has incomplete type
/sw/slc8_x86-64/fmt/10.1.1-1/include/fmt/core.h:1580:7: error: static assertion failed: Cannot format an argument. To make type T formattable provide a formatter<T> specialization: https://fmt.dev/latest/api.html#udt
ninja: build stopped: subcommand failed.
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/12298/789f8a0b64e690a4c090fed621ceeb586ac04dc6/build_O2_fullCI/pretty.html).

--- a/Framework/Core/src/DataSpecUtils.cxx
+++ b/Framework/Core/src/DataSpecUtils.cxx
@@ -13,6 +13,7 @@
 #include "Framework/DataMatcherWalker.h"
 #include "Framework/VariantHelpers.h"
 #include "Framework/RuntimeError.h"
+#include "Headers/DataHeaderHelpers.h"
 
 #include <fmt/format.h>
 #include <cstring>

